bitnami/ejbca/values.yaml

```diff
@@ -0,0 +1,354 @@
+## Global Docker image parameters
+## Please, note that this will override the image parameters, including dependencies, configured to use the global value
+## Current available global Docker image parameters: imageRegistry and imagePullSecrets
+##
+# global:
+#   imageRegistry: myRegistryName
+#   imagePullSecrets:
+#     - myRegistryKeySecretName
+#   storageClass: myStorageClass
+
+## Bitnami EJBCA image version
+## ref: https://hub.docker.com/r/bitnami/ejbca/tags/
+##
+image:
+  registry: docker.io
+  repository: bitnami/ejbca
+  tag: latest
+  ## Specify a imagePullPolicy
+  ## Defaults to 'Always' if image tag is 'latest', else set to 'IfNotPresent'
+  ## ref: http://kubernetes.io/docs/user-guide/images/#pre-pulling-images
+  ##
+  pullPolicy: IfNotPresent
+  ## Optionally specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  ##
+  # pullSecrets:
+  #   - myRegistryKeySecretName
+  ## Set to true if you would like to see extra information on logs
+  ##
+
+## String to partially override ebjca.fullname template (will maintain the release name)
+##
+# nameOverride:
+
+## String to fully override ebjca.fullname template
+##
+# fullnameOverride:
+
+replicaCount: 1
```

Comment:
I would add here a comment, as the rest of parameters has their own comment this looks weird

Reply:
Done! Thanks for the heads-up